GCRCatalogs/site_config/site_rootdir.yaml

```diff
@@ -0,0 +1,2 @@
+nersc:  '/global/projecta/projectdirs/lsst/groups/CI/gcr_rootdir_test'
+in2p3:  '/sps/lsst/data/desc/gcr_rootdir'
```

Comment:
```suggestion
nersc: '/global/projecta/projectdirs/lsst/groups/CI/gcr_rootdir_test'
in2p3: '/sps/lsst/data/desc/gcr_rootdir'
```

Reply:
It's hard to tell from the "suggested  change" display above, but I *think* you're asking that the two spaces following : be changed to a single space.   I've made that change.  If you want all yaml to conform to a very specific style like this I suggest you document it somewhere in the package (or is it already there and I've missed it?)

Reply:
Yes I was removing the extra space. It's not a big deal though. We don't enforce a strict formatting rule and I removed them only because I happen to notice them.